packages/telemetry/src/slog-to-otel.js

```diff
@@ -571,52 +751,48 @@ export const makeSlogToOtelKit = (tracer, overrideAttrs = {}) => {
         }
         break;
       }
-      case 'bundle-kernel-start': {
-        spans.push(`bundle-kernel`);
-        break;
-      }
-      case 'bundle-kernel-finish': {
-        spans.pop('bundle-kernel');
+      case 'kernel-stats': {
+        const stats = /** @type {SpanAttributes} */ ({});
+        Object.entries(slogAttrs.stats).forEach(
```

Comment:
Why not use `cleanAttrs` here instead, or at least `serializeInto` if the goal was to avoid the potential `overrideAttrs` ?